presentation/frontend/src/test/favorite-properties/FavoritePropertiesStorage.test.ts

```diff
@@ -132,8 +132,7 @@ describe("IModelAppFavoritePropertiesStorage", () => {
         setting: [orderInfo],
       }));
 
-      // SWB
-      const properties = await storage.loadPropertiesOrder("projectId", "imodelId");
+      const properties = await storage.loadPropertiesOrder("itwinId", "imodelId");
```

Comment:
Why are itwinId and imodelId breaking convention with lowercase?

Reply:
@calebmshafer @grigasp What are the implications of this hardcoded string?

Reply:
none, they just test strings here